app/views/customers/show.html.erb

```diff
@@ -8,3 +32,17 @@
 
   <%= button_to "Destroy this customer", @customer, method: :delete %>
 </div>
+
+<h1>Customer: <%= @customer.full_name %></h1>
+
+<h2>Orders:</h2>
+<% if @orders.any? %>
+  <ul>
+    <% @orders.each do |order| %>
+      <li><%= order.description %></li>
```

Comment:
The orders pages aren't implemented/broken, so I had to go into the Rails console to manually associate a new order with a customer. However, that reveals a fatal bug:

![Screenshot 2024-09-07 at 10 36 31 PM](https://github.com/user-attachments/assets/0591bad6-da47-4bb7-8953-693cf56a3310)